### DISCUSSION TITLE
Cannot use llama2-13b-chat-german.ggmlv3.q4_0 model

### DISCUSSION THREAD

**Discussion by Lue-C** [imartinez/privateGPT#1048] [Q&A]
Hi,
I want to use the "llama2-13b-chat-german.ggmlv3.q4_0". I downloaded the file from https://huggingface.co/jphme/Llama-2-13b-chat-german-GGML/tree/main, put it in the models folder and ran

`python privategpt.py`

The following error message appeared:

`File "C:\Users\mlu\Repositories\vis-gpt\privateGPT.py", line 106, in <module>
    main()
  File "C:\Users\mlu\Repositories\vis-gpt\privateGPT.py", line 45, in main
    llm = LlamaCpp(model_path=model_path, max_tokens=model_n_ctx, n_batch=model_n_batch, callbacks=callbacks, verbose=False)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mlu\Repositories\vis-gpt\.vis_gpt_venv\Lib\site-packages\langchain\load\serializable.py", line 74, in __init__
    super().__init__(**kwargs)
  File "pydantic\main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for LlamaCpp
__root__
  Could not load Llama model from path: models/llama2-13b-chat-german.ggmlv3.q4_0.bin. Received error  (type=value_error)`

The package versions are

`langchain==0.0.274
gpt4all==1.0.8
chromadb==0.4.7
llama-cpp-python==0.1.81
urllib3==2.0.4
PyMuPDF==1.23.1
python-dotenv==1.0.0
unstructured==0.10.8
extract-msg==0.45.0`

Does anyone know why this error occures or know a solution?

Thanks in advance

**Comment by Hempstead** [comment]
I have similar issue with 7B model

**Comment by FW-Treeman** [comment]
same issue here :(

**Comment by rhoughton98** [comment]
I have a similar problem too
